requirements_dev.txt

```diff
@@ -1,13 +1,13 @@
 -r requirements.txt
-black==21.4b2
+black==21.5b0
 coverage==5.5
 coveralls==3.0.1
-flake8==3.9.1
-myst-parser==0.13.7
+flake8==3.9.2
+myst-parser==0.14.0
 pre-commit==2.12.1
-pytest==6.2.3
+pytest==6.2.4
 pytest-cov==2.11.1
-Sphinx==3.5.4
+Sphinx==3.5.4      # pin this until Myst is happy with Sphinx 4
```

Comment:
@Br4nd0R could you help Paul get snowfakery switched to pip-tools so we have a better way to express this kind of thing?